siriuspy/siriuspy/sofb/orbit.py

```diff
@@ -22,6 +22,23 @@
 from .bpms import BPM, TimingConfig, TIMEOUT
 
 
+# NOTE: I have to rederive epics.CAProcess here to ensure the process will be
+# launched with the spawn method.
+class _Process(_mp.get_context('spawn').Process):
+    """
+    A Channel-Access aware (and safe) subclass of multiprocessing.Process
+    Use CAProcess in place of multiprocessing.Process if your Process will
+    be doing CA calls!
+    """
+    def __init__(self, **kws):
+        _mp.Process.__init__(self, **kws)
+
+    def run(self):
+        _ca.initial_context = None
+        _ca.clear_cache()
+        _mp.Process.run(self)
+
+
```

Comment:
It would be interesting to define a kind of "ProcessSpawn" class in the siriuspy.epics subpackage to avoid repeated derivations, wouldn't it?

Reply:
nice @anacso17 ! I'll do it"

Reply:
done!